client/packages/invoices/src/InboundShipment/DetailView/SidePanel/PricingSection.tsx

```diff
@@ -59,19 +94,29 @@ export const PricingSectionComponent = () => {
           <PanelLabel>{t('heading.sub-total')}</PanelLabel>
           <PanelField>{c(pricing.serviceTotalBeforeTax).format()}</PanelField>
         </PanelRow>
+        <PanelRow>
+          <PanelLabel>{`${t('heading.tax')} ${Formatter.tax(tax)}`}</PanelLabel>
```

Comment:
Same heading as the above row, but different values (tax & percentage).. should one heading say 'percentage'?

Reply:
One is for stock charge and the other is for service charge (: